README.md

```diff
@@ -749,6 +749,55 @@ public string FunctionHandler(string input, ILambdaContext context)
 }
 ```
 
+### Oversampling Mitigation
+Oversampling mitigation allows you to ignore a parent segment/subsegment's sampled flag and instead set it to false.
```

Comment:
Same comment as Go PR:
Can this explanation of oversampling mitigation be reworded to be more clear? I think it is important to clarify what it refers to in "instead set it to false" and also explain how the subsegments now inherit the sampling decision of their direct parent - segment or subsegment - as opposed to always referring to the parent segment.